hebiros_advanced_examples/launch/mapping_navigation/omnibase_navigation.launch

```diff
@@ -0,0 +1,46 @@
+<launch>
+  <!-- Arguments -->
+  <arg name="map_file" default="$(find hebiros_advanced_examples)/launch/mapping_navigation/maps/Hebi_map.yaml"/>
+  <arg name="open_rviz" default="true"/>
+  <arg name="x_pos" default="0.0"/>
+  <arg name="y_pos" default="0.0"/>
+  <arg name="z_pos" default="0.0"/>
+
+
+
+  <!-- Map server -->
+  <node pkg="map_server" name="map_server" type="map_server" args="$(arg map_file)"/>
+
+  <!-- AMCL -->
+  <include file="$(find hebiros_advanced_examples)/launch/mapping_navigation/amcl.launch"/>
+
+  <!-- move_base -->
+  <include file="$(find hebiros_advanced_examples)/launch/mapping_navigation/move_base.launch"/>
+
+
+<!--RP Lidar-->
+  <node name="rplidarNode"          pkg="rplidar_ros"  type="rplidarNode" output="screen">
+  <param name="serial_port"         type="string" value="/dev/ttyUSB0"/>  
+  <param name="serial_baudrate"     type="int"    value="256000"/><!--A3 -->
+  <param name="frame_id"            type="string" value="base_scan"/>
+  <param name="inverted"            type="bool"   value="false"/>
+  <param name="angle_compensate"    type="bool"   value="true"/>
+  <param name="scan_mode"           type="string" value="Sensitivity"/>
+  </node>
+
+  <!-- rviz -->
+  <group if="$(arg open_rviz)"> 
+    <node pkg="rviz" type="rviz" name="rviz" required="true"
+          args="-d $(find hebiros_advanced_examples)/rviz/omnibase_navigation.rviz"/>
+  </group>
+
+  <node name="robot_state_publisher" pkg="robot_state_publisher" type="robot_state_publisher"
+    respawn="false" output="screen"/>
+
+  <param name="robot_description" command="$(find xacro)/xacro --inorder $(find hebiros_description)/urdf/omnibase.xacro" />
+
+  <!--node pkg="gazebo_ros" type="spawn_model" name="spawn_urdf"  args="-urdf -model omnibase -x $(arg x_pos) -y $(arg y_pos) -z $(arg z_pos) -param robot_description" /-->  
```

Comment:
I noticed this the other day -- is this commented out line useful for something?  Are there issues setting non-zero initial positions?  If so, perhaps we remove the arguments at the top of the file, too?